tests/storage/iso/test_local_iso_sr.py

```diff
@@ -5,6 +5,7 @@
     check_iso_mount_and_read_from_vm, remove_iso_from_sr
 
 # Requirements:
+# A shared SR or a default SR on the master
 # From --hosts parameter:
 # - host: a XCP-ng host
```

Comment:
It's a bit awkward to mention the shared SR before even mentioning the host/pool.

```suggestion
# From --hosts parameter:
# - host: a XCP-ng host, with the default SR being either a shared SR, or a local SR on the master host
```